UnityProject/Assets/Scripts/Factories/EffectsFactory.cs

```diff
@@ -71,8 +74,7 @@ public GameObject SpawnShroudTile(Vector3 pos)
 	[Server]
 	public void BloodSplat(Vector3 pos, BloodSplatSize splatSize)
 	{
-		GameObject b = PoolManager.Instance.PoolNetworkInstantiate(bloodTile, pos, Quaternion.identity, 
-			MatrixManager.AtPoint( Vector3Int.RoundToInt( pos ) ).GameObject.transform);
```

Comment:
`MatrixManager.AtPoint( Vector3Int.RoundToInt( pos ) ).GameObject.transform` this is the parent that the blood splat needs to be a child of. So removing it would make the blood splat not a child of anything. If the blood was in a shuttle then the shuttle would fly off and leave the blood behind. Maybe try setting the parent on a separate line after it has instantiated and see what happens.

so: `b.parent = MatrixManager.AtPoint( Vector3Int.RoundToInt( pos ) ).GameObject.transform;`